drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
Interesting. I never though about that. So you suggest that using a hydra client to `GET` the documentation URI should give you state of the entrypoint? If so, I'd mention that in the `ApiDocumentation` page